spec/lib/annotate/annotate_models_spec.rb

```diff
@@ -1631,6 +1631,16 @@ def mock_column(name, type, options = {})
         end
       end
     end
+
+    context 'when pattern_type is "admin"' do
+      let(:root_directory) { '/root' }
+      let(:pattern_type) { 'admin' }
+      let(:options) { {} }
+
+      it 'returns an empty array' do
```

Comment:
Is this expectation description accurate?

Reply:
I think you're right. I've updated the description. Thanks!